modules/organization/src/endpoints/mod.rs

```diff
@@ -0,0 +1,75 @@
+use crate::service::OrganizationService;
+use actix_web::{get, web, HttpResponse, Responder};
+use trustify_common::db::query::Query;
+use trustify_common::db::Database;
+use trustify_common::model::Paginated;
+use utoipa::OpenApi;
+
+pub fn configure(config: &mut web::ServiceConfig, db: Database) {
+    let service = OrganizationService::new(db);
+    config
+        .app_data(web::Data::new(service))
+        .service(web::scope("/api/v1/organization").service(all).service(get));
```

Comment:
I'm not sure exactly what the right way is, but I wish we could come up with a standard convention for arranging endpoints. I know we had issues with `web::scope` with advisories, so I might suggest we remove this and...

Reply:
I think the new modularization "solves" the web::scope issue for now.

You can ask the organization module to weave itself in. It's the only thing that can own /api/v1/organization, and can choose to /api/v2.3/organization also w/o having to muck with the server end of the stick.